pooch/core.py

```diff
@@ -467,7 +474,7 @@ def load_registry(self, fname):
 
         Use this if you are managing many files.
 
-        Each line of the file should have file name and its SHA256 hash
+        Each line of the file should have file name and its hash ("alg:checksum")
         separate by a space. Only one file per line is allowed. Custom download
         URLs for individual files can be specified as a third element on the
         line.
```

Comment:
This would also be a good place to say that the `alg:checkum` syntax is optional and sha256 will be used by default.

Reply:
Fixed in 0a5e712